drafts/19008-security.adoc

```diff
@@ -8,76 +8,103 @@ seo-title: Securing Open Liberty apps and microservices - OpenLiberty.io
-
-Developers can also configure applications so that, after the user has logged in once, they do not have to log in again to every application separately. Opne Liberty provides a SAML client (`samlWeb-2.0`) for use with a SAML identity provider and a Spnego client (`spnego-1.0`) for use with Microsoft domain controllers.
-
+It is also possible to configure Open Liberty as an OpenID Connect provider. The OpenID Connect Server (`openidConnectServer-1.0`) feature can be configured to manage user authentication and to issue JWTs for use by OpenID Connect clients and microservices. The OpenID Connect Server can work directly with a user registry such as LDAP or act as an identity broker and delegate to another security provider.
 
 //
 
 == Example: Using OpenID Connect and JWTs to authenticate users and secure microservices:
 
-The following sample code and configurations use three servers to demonstrate authenticating a user and accessing a secured microservice using a JWT. The three pieces of this are:
+The following sample code and configurations use two servers to demonstrate authenticating a user, building a JWT and accessing a secured microservice using that JWT. The pieces of this are:
 
-* A server called `oidc_client` with the OpenID Connect Client feature (`openIdConnectClient-1.0`) hosts a servlet app secured with OpenID Connect.  
+* A server called `oidc_client` with the Social Login feature (`socialLogin-1.0`) hosts a servlet app secured with OpenID Connect. Google is the designated security provider.  
 +
 * A server called `microservice` with the MicroProfile JWT feature (`mpJwt-1.1`) hosts a JAX-RS microservice that is secured with MicroProfile JWT.
-+
-* A server called `oidc_provider` with the OpenID Connect Server feature (`openidConnectServer-1.0`) provides user authentication for the OpenID Connect client.
 
-When a user attempts to access the servlet, their browser is redirected to the OpenID Connect Provider for them to enter their ID and password.  Upon success, the browser gets redirected back to the client with an authorization code.  The client then contacts the provider, obtains a JWT access token, then uses it to create an authenticated user on the client, which then gets access to the servlet.
+An application was first registered with Google, which then supplied the client ID and client secret used in server.xml.
+(Registration instructions are https://developers.google.com/identity/protocols/OAuth2[here] ).
+
+The public certificate from Google was added to the client, and certificates exchanged between client and microservice as described https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_add_trust_cert.html[here].
```

Comment:
"The public certificate from Google is added to the client, and certificates exchanged between client and microservice as described"